types/koa-docs/index.d.ts

```diff
@@ -0,0 +1,28 @@
+// Type definitions for koa-docs 2.1
+// Project: https://github.com/a-s-o/koa-docs#readme
+// Definitions by: 4doge <https://github.com/4doge>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 2.4
+
+import { Middleware } from 'koa';
+import { Spec } from 'koa-joi-router';
+
+export {};
+
+declare namespace koaDocs {
```

Comment:
Why bother with the `koaDocs` namespace and the `export {}` - why not just `export interface Options` (making the options interface public)?